staging/src/k8s.io/apiserver/pkg/util/flowcontrol/request/width.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package request
+
+import (
+	"net/http"
+)
+
+// NewWidthEstimator returns a WidthEstimatorFunc
+// that returns '1' as the "width" of the given request.
+//
+// TODO: when we plumb in actual "width" handling for different
+//  type of request(s) this function will iterate through a chain
+//  of widthEstimator instance(s).
+func NewWidthEstimator() WidthEstimatorFunc {
```

Comment:
I do not see why to have a function that returns a new WidthEstimatorFunc, rather than simple declare one WidthEstimatorFunc.

Reply:
+1

Reply:
done